src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Tables/ResourceChangeData.sql

```diff
@@ -6,6 +6,8 @@ CREATE TABLE dbo.ResourceChangeData
     ResourceId varchar(64) NOT NULL,
     ResourceTypeId smallint NOT NULL,
     ResourceVersion int NOT NULL,
-    ResourceChangeTypeId tinyint NOT NULL,
-    CONSTRAINT PK_ResourceChangeData_TimestampId PRIMARY KEY (Timestamp, Id)
```

Comment:
Suggestion:  Might be good to explain why we are making this change:
* This allows us to remove the partition column from the PK which leads to better query performance.
*  Uniqueness was only per partition before, i.e. same key could occur in different partitions.
*  We are using identity which will guarantee uniqueness unless identity insert is used on the table which shouldn't happen.
* To enforce global uniqueness requires a non clustered index without a partition which prevents partition swaps.

Reply:
Added a comment on the index creation in a ResourceChangeData table's SQL file, and diff file.